rbac/api/cross_access/view.py

```diff
@@ -262,7 +253,13 @@ def update_status(self, car, status):
         car.status = status
         if status == "approved":
             create_cross_principal(car.user_id, target_org=car.target_org)
-
+            cross_account_roles = car.roles.all()
+            if any(True for _ in cross_account_roles):
```

Comment:
A bit of a nit, but I think it would be more performant (assuming I'm understanding the intent here) to do something like this with [`exists()`](https://docs.djangoproject.com/en/4.2/ref/models/querysets/#django.db.models.query.QuerySet.exists):

```suggestion
            if cross_account_roles.exists():
```

Using `any()` _will_ return early, but it executes the query on the queryset which in this case is a select on all roles for a car, where `exists()` will limit the query to a single row.